crates/proto/src/xfer/dns_handle.rs

```diff
@@ -71,7 +71,7 @@ fn build_request(mut query: Query, options: DnsRequestOptions) -> DnsRequest {
     let mut message: Message = Message::new();
     // TODO: This is not the final ID, it's actually set in the poll method of DNS future
     //  should we just remove this?
-    let id: u16 = rand::random();
+    let id: u16 = random();
```

Comment:
Actually, note the `TODO` above. Can you check that and see if we can just remove this line?

Reply:
Since we don't even use the `poll` method in no_std, I don't think the TODO would apply to all use cases 🤔